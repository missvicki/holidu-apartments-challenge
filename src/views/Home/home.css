#page-container {
    position: relative;
    min-height: 100vh;
}
  
#content {
    padding-bottom: 2.5rem;    
    margin-bottom: 3rem;
}