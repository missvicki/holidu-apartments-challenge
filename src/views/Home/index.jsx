import React from 'react';
import Header from '../../shared/Header';
import Footer from '../../shared/Footer';
import Main from '../../containers/Main';
import './home.css';

function Home(){
    return(
        <div className="page-container">
            <div id="content">
                <Header />
                <Main />
            </div>
            <Footer />
        </div>
    )
}

export default Home;