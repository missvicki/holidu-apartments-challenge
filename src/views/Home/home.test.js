import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

describe('Home', () => {
  it('should not have regressed', () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
