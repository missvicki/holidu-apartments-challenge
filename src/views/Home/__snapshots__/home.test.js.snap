// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should not have regressed 1`] = `
<div
  className="page-container"
>
  <div
    id="content"
  >
    <Header />
    <Connect(Main) />
  </div>
  <Footer />
</div>
`;
